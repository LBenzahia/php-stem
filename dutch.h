
/* This file was generated automatically by the Snowball to ANSI C compiler */

extern struct SN_env * dutch_create_env(void);
extern void dutch_close_env(struct SN_env * z);

extern int dutch_stem(struct SN_env * z);

