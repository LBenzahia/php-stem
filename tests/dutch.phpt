--TEST--
Dutch stemming algorithm (extensive)
--SKIPIF--
<?php if (!extension_loaded("stem") || !stem_enabled(STEM_DUTCH)) print "skip"; ?>
--POST--
--GET--
--FILE--
<?php

require("fulltests.php");

runTest(STEM_DUTCH, "dutch.txt");

?>
--EXPECT--
BAD: 0
