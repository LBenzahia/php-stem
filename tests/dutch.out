a
a
Ã 
aa
aach
aachener
aah
aalborg
aalder
aalmoez
aalscholver
aalscholver
aalscholverstand
aalsmer
aalsmeerban
aalsmer
aalt
aaltenar
aaltjes
aam
aan
aan
aanbad
aanbaklaagjes
aanbeld
aanbell
aanbested
aanbested
aanbested
aanbestedingsdossier
aanbestedingsprocedur
aanbestedingsrichtlijn
aanbevdlijn
awlijn
awloijn
ijn
ijn
ijn
ijn
ivn
ijnsni